io/net/bridge.py

```diff
@@ -82,20 +83,19 @@ def test_bridge_run(self):
                                             if_type="Bridge")
         try:
             networkinterface.add_ipaddr(self.ipaddr, self.netmask)
-            networkinterface.save(self.ipaddr, self.netmask)
         except Exception:
             networkinterface.save(self.ipaddr, self.netmask)
         networkinterface.bring_up()
         if networkinterface.ping_check(self.peer_ip, count=5) is not None:
             self.fail('Ping using bridge failed')
         networkinterface.remove_ipaddr(self.ipaddr, self.netmask)
+        try:
+            networkinterface.restore_from_backup()
```

Comment:
@vaishnavibhat  this is not needed for bridge interface,why need to restore cfg file for bridge interface ..?

Reply:
Used this for removing any cfg files that are created because we are using networkinterface.save for bridge. 
try:
            networkinterface.add_ipaddr(self.ipaddr, self.netmask)
        except Exception:
            networkinterface.save(self.ipaddr, self.netmask)

Reply:
@vaishnavibhat  restore_from_backup() this will not remove the cfg files, it will just restore the .backup file to cfg file.

Reply:
resent the patch after removing restore_from_backup(). 
This function may not be required because the patch removes the usage of  networkinterface.save(self.ipaddr, self.netmask) which saves the cfg file.